### PR TITLE
[Fix] hit rate too low for victim cache

### PR DESCRIPTION
**Pull request by jackyhuang85**

### CODE REVIEW

File: src/cache.cpp [hunk 1]
```diff
@@ -10,13 +12,19 @@ Cache::Cache(const CACHE_SET &cfg)
     _cache_setting = cfg;
     _Cache_Setup();
 }
-
+Cache::Cache()
```
Comment (jerryzj) [inline src/cache.cpp]: Codacy Issue found: [Member variable 'Cache::_has_evicted' is not initialized in the constructor.](https://app.codacy.com/app/jerry0715no14/Cache_sim/pullRequest?prid=4646302)

**Comment by jerryzj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/jerry0715no14/Cache_sim/pullRequest?prid=4646302&bid=15560825)

File: src/simulator.cpp [hunk 2]
```diff
@@ -241,11 +336,80 @@ void Simulator::DumpResult() {
...
-    std::cout << "\n";
-    std::cout << "Number of cache access： " << _counter.access << std::endl;
-    std::cout << "Number of cache load： " << _counter.load << std::endl;
-    std::cout << "Number of cache store： " << _counter.store << std::endl;
-    std::cout << "Cache hit rate: " << std::setprecision(6)
-              << _counter.avg_hit_rate << std::endl;
-    std::cout << "===================================" << std::endl;
 }
+
+void Simulator::_DumpCACTIConfig(const std::string filename,
```
Comment (jerryzj) [inline src/simulator.cpp]: Codacy Issue found: [Function parameter 'filename' should be passed by const reference.](https://app.codacy.com/app/jerry0715no14/Cache_sim/pullRequest?prid=4646302)

**Comment by jerryzj** [comment]
@yaohsiaopid @jackyhuang85 , please fix the issues found by PR Quality Review.
Thanks.

**Comment by jackyhuang85** [comment]
@jerryzj  Done!

**Pull request merged**